repository/test-src/org/pentaho/platform/repository2/unified/webservices/jaxws/UnifiedRepositoryToWebServiceAdapterIT.java

```diff
@@ -75,7 +78,7 @@ public void testFileMetadata() throws Exception {
       assertNotNull( testfile.getId() );
       final Map<String, Serializable> fileMetadata = repo.getFileMetadata( testfile.getId() );
       assertNotNull( fileMetadata );
-      assertEquals( 0, fileMetadata.size() );
+      assertEquals( 1, fileMetadata.size() );
```

Comment:
Please explain the change in the test case here. Ordinarily, this would set off red flags - we have a working test where we expect the fileMetadata size to be zero. After your change, the size is no longer zero. Why has this changed, and what possible affect may the change have downstream ? Please explain any such change to an existing test case in a lengthy comment.